app/uk/gov/hmrc/cataloguefrontend/service/CostEstimationService.scala

```diff
@@ -58,21 +61,65 @@ object CostEstimationService {
       Json.reads[DeploymentConfig]
   }
 
-  final case class CostEstimation(totalSlots: Int) {
+  final case class ServiceCostEstimate(forEnvironments: List[ServiceCostEstimate.ForEnvironment]) {
```

Comment:
Would `Map[Environment,Summary]` be more practical? (we can be confident we only one for each etc.)

Reply:
I'll have a tinker, but suspect I'll only ever be operating on the `.values` projection if I were to do this. The one for each point is persuasive though 👍

Reply:
`.values` or `.toList`, that is